src/components/tabs/chapters/ChapterEntry.vue

```diff
@@ -147,15 +161,20 @@
         },
 
         onChapterPlayClick: function ({ dispatch, actions }, event) {
+          if (event.target.classList.contains('info-link')) {
```

Comment:
instead of preventing the click event on the `<link-icon>` I would recommend to move the `onChapterPlayClick` directly to the `<play-icon>`.

Reply:
Since I really don't want to influence the general usability negatively by just making the play click work on the play symbol, i stuck with this scheme. But instead of inspecting the target, I'm susing the linkHover bool now to make that decision.